commercetools-test-lib/src/main/java/io/sphere/sdk/test/IntegrationTest.java

```diff
@@ -111,7 +120,21 @@ private static SphereClient withMaybeDeprecationWarnTool(final SphereClient unde
 
     protected static HttpClient newHttpClient() {
         //NO SSL Client: this client doesn't perform ssl certification check, which is a necessity to run tests on CI
-        return IntegrationTestHttpClient.of(HttpAsyncClients.custom().setSSLHostnameVerifier(NoopHostnameVerifier.INSTANCE).build());
+        return IntegrationTestHttpClient.of(createNoSSLClient());
+    }
+
+    private static CloseableHttpAsyncClient createNoSSLClient() {
+        TrustStrategy acceptingTrustStrategy = (X509Certificate[] chain, String authType) ->  true;
+        SSLContext sslContext = null;
+        try {
+            sslContext = SSLContexts.custom().loadTrustMaterial(null, acceptingTrustStrategy).build();
+        } catch (Exception e) {
+            logger.error("Could not create SSLContext");
```

Comment:
Please log the exception, otherwise we will have a hard time debugging the next issue 😉

Reply:
👍 done!